server/app/app.go

```diff
@@ -320,7 +318,7 @@ func registerOtlpServer(app *App, testDB model.Repository, eventEmitter executor
 	})
 }
 
-func registerlinterResource(linterRepo *linterResource.Repository, router *mux.Router, db *sql.DB, provisioner *provisioning.Provisioner, tracer trace.Tracer) {
+func registerlinterResource(linterRepo *linterResource.Repository, router *mux.Router, provisioner *provisioning.Provisioner, tracer trace.Tracer) {
```

Comment:
the `L` should be uppercase here
```suggestion
func registerLinterResource(linterRepo *linterResource.Repository, router *mux.Router, provisioner *provisioning.Provisioner, tracer trace.Tracer) {
```